panther_description/urdf/panther_macro.urdf.xacro

```diff
@@ -44,26 +45,30 @@
 
     <!-- WHEEL DECLARATION -->
     <xacro:wheel.wheel
+      namespace="${namespace}"
       wheel_config="${wheel_config}"
       wheel_separation_x="${wheel_separation_x}"
       prefix="fl" />
 
     <xacro:wheel.wheel
+      namespace="${namespace}"
       wheel_config="${wheel_config}"
       wheel_separation_x="${wheel_separation_x}"
       prefix="fr" />
 
     <xacro:wheel.wheel
+      namespace="${namespace}"
       wheel_config="${wheel_config}"
       wheel_separation_x="${wheel_separation_x}"
       prefix="rl" />
 
     <xacro:wheel.wheel
+      namespace="${namespace}"
       wheel_config="${wheel_config}"
       wheel_separation_x="${wheel_separation_x}"
       prefix="rr" />
 
-    <ros2_control name="${ns}wheels" type="system">
+    <ros2_control name="${ns}interfaces" type="system">
```

Comment:
`interfaces` is not a good name. The imu also has interfaces. Moreover, not only interfaces  but also the hardware plugins  are declared here. I would name this `panther_base_system` or `system` or leave `wheels`.